ci/plugins/mzcompose/hooks/command

```diff
@@ -18,7 +18,8 @@ mzcompose() {
 service=$BUILDKITE_PLUGIN_MZCOMPOSE_RUN
 run_args=("$service")
 if [[ "${BUILDKITE_COMMAND:-}" ]]; then
-    run_args+=("$BUILDKITE_COMMAND")
+    # shellcheck disable=SC2206
+    run_args+=($BUILDKITE_COMMAND)
```

Comment:
why are you taking this out of quotes? Anytime SC2206 is disabled there should be a comment on the line above explaining why we want word splitting here.

Reply:
Apparently I added this. Can't remember why. Pushed up a change without it to see what happens.